Source/Core/DolphinQt/Settings/PathPane.h

```diff
@@ -37,5 +37,6 @@ class PathPane final : public QWidget
   QLineEdit* m_dump_edit;
   QLineEdit* m_sdcard_edit;
 
+  QPushButton* add;
```

Comment:
Likely irrelevant as I think the previous comment will be the proper fix but our code-style says private variables should be prefixed with m_.